app/components/Poll/index.js

```diff
@@ -64,17 +64,39 @@ class Poll extends React.Component<Props, State> {
       : this.setState({ optionsToShow: options, expanded: true });
   };
 
+  optionsWithRatios = (options, optionsToShow) => {
+    const totalVotes = options.reduce((a, option) => a + option.votes, 0);
+    return optionsToShow.map(option => {
+      return {...option, "ratio": option.votes / totalVotes * 100}
+    });
+  };
+
+  // As described in: https://stackoverflow.com/questions/13483430/how-to-make-rounded-percentages-add-up-to-100
+  perfectRatios = (options) => {
+    const off = 100 - options.reduce((a, option) => a + Math.floor(option.ratio), 0);
+    return options
+        .sort((a, b) => {
+          const aMantissa = a.ratio - Math.floor(a.ratio);
+          const bMantissa = b.ratio - Math.floor(b.ratio);
+          return aMantissa < bMantissa;
+        })
+        .map((option, index) => {
+          return {...option, "ratio": Math.floor(option.ratio) + (index < off ? 1 : 0)}
+        })
+        .sort((a, b) => b.ratio - a.ratio)
```

Comment:
The end result is sorted on ratio with the biggest ratio being the uppermost option in the poll. Maybe you would prefer to sort on something else?

Reply:
Seems logical to me